package.json

```diff
@@ -1,163 +1,163 @@
-{

```

Comment:
Please revert to original package.json...

This version has too many changes ( probably necessarry EOL  or different tab space convention)

Reply:
For the future it should be considered to add a .gitattributes file, to have a consistent line ending which is not depending on the contributor.
See: https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings